data-raw/family.R

```diff
@@ -0,0 +1,15 @@
+library(dplyr)
+
+family <- tribble(
+  ~family,  ~dob_child1,  ~dob_child2, ~name_child1, ~name_child2,
+        1, "1998-11-26", "2000-01-29",      "Susan",       "Jose",
+        2, "1996-06-22",           NA,       "Mark",           NA,
+        3, "2002-07-11", "2004-04-05",        "Sam",       "Seth",
+        4, "2004-10-10", "2009-08-27",      "Craig",       "Khai",
+        5, "2000-12-05", "2005-02-28",     "Parker",     "Gracie",
```

Comment:
I noticed that `family` is a double column, not sure if you wanted an integer column. Probably doesn't matter but thought id point it out